src/app/components/searchbar/searchbar.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-searchbar',
+  templateUrl: './searchbar.component.html',
+  styleUrls: ['./searchbar.component.css']
+})
+export class SearchbarComponent implements OnInit {
+
+  @Output() searched = new EventEmitter();
```

Comment:
This name should be more aligned with event names. No past tense or anything. It could be just `search` which will be read as event `on search`